crd2pulumi-crontabs/README.md

```diff
@@ -0,0 +1,233 @@
+$ tar -xvf crd2pulumi-darwin-amd64.tar.gz
+$ mv ./releases/crd2pulumi-darwin-amd64/crd2pulumi ./crd2pulumi
+```
+
+## Running the App (TypeScript)
+
+Follow the steps in [Pulumi Installation and Setup](https://www.pulumi.com/docs/get-started/install/) and [Configuring Pulumi
+Kubernetes](https://www.pulumi.com/docs/intro/cloud-providers/kubernetes/setup/) to get setup with Pulumi and Kubernetes.
+
+Install dependencies:
+
+```bash
+$ cd kubernetes-ts-crontabs
+$ npm install
+```
+
+Create a new stack:
+
+```bash
+$ pulumi stack init dev
+```
+
+At first, the provided `index.ts` program shouldn't run, since we haven't actually generated the `./crontabs` code yet.
+
+> `resourcedefinition.yaml` is a k8s CRD that specifies a CronTab CustomResource. It's also used as an example in the [Kubernetes Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/).
+
+Generate the strongly-typed CronTab resource in the current directory:
+```bash
+$ ../crd2pulumi nodejs ../resourcedefinition.yaml .
+```
+
+This should generate a `./crontabs` folder, where we can import the useful classes `v1.CronTab` and `NewCronTabDefinition`.
+> This saves us a lot of time, since we can create the CRD in a single line and get strongly-typed arguments in the CustomResource. If you're curious, the comments contain the code that we would've written without `crd2pulumi`.
+
+Perform the deployment:
+```bash
+$ pulumi up
+```
+```bash
+Previewing update (dev):
+ 	Type                                                     	Name               	Plan
+ 	pulumi:pulumi:Stack                                      	examples-dev
+ +   ├─ kubernetes:stable.example.com:CronTab                 	my-new-cron-object 	create
+ +   └─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  my-crontab-definition  create
+Resources:
+	+ 2 to create
+	1 unchanged
+Do you want to perform this update?  [Use arrows to move, enter to select, type to filter]
+> yes
+  no
+  details
+```
+```bash
+Do you want to perform this update? yes
+Updating (dev):
+ 	Type                                                     	Name               	Status
+ 	pulumi:pulumi:Stack                                      	examples-dev
+ +   ├─ kubernetes:stable.example.com:CronTab                 	my-new-cron-object 	created
+ +   └─ kubernetes:apiextensions.k8s.io:CustomResourceDefinition  my-crontab-definition  created
+Outputs:
+	urn: "urn:pulumi:dev::examples::kubernetes:stable.example.com/v1:CronTab::my-new-cron-object"
+Resources:
+	+ 2 created
+	1 unchanged
+Duration: 17s
+Permalink: https://app.pulumi.com/albert-zhong/examples/dev/updates/4
+```
+
+It looks like both the CronTab definition and instance were both created! Finally, let's verify that they were created
+by manually viewing the raw YAML data:
+
+```bash
+$ kubectl get ct -o yaml
+```
+
+```bash
```

Comment:
I realize this is the output from the shell, so "bash" is technically correct, but since you have it as a separate code block, wondering if it'd render nicer as "yaml".

```suggestion
```yaml
```